app/models/stenographer/change.rb

```diff
@@ -21,8 +21,9 @@ def sanitize_environment
       end
     end
 
-    def to_markdown
-      Stenographer.markdown_renderer.render(message).html_safe
+    def to_markdown(truncate: nil)
+      msg = truncate ? message.truncate(truncate) : message
```

Comment:
would the truncation cause invalid markdown? or the markdown_renderer would take care of it?

Reply:
the renderer should handle it. like

[foobar](https://...

for example would just stay as-is